### ISSUE TITLE
[FEATURE]

### ISSUE THREAD

**Issue by rogerct** [Tiggy-Dev/aniflix-site#3]
Es wäre super, wenn ihr animes bei dub und sub sortieren könntet!

**Comment by Tiggy-Dev** [comment]
Du meinst in der Suchfunktion ? 
oder wie meinst du das mit dem sortieren?

**Comment by rogerct** [comment]
Ja, in der Suchfunktion oder vielleicht  bei https://www2.aniflix.tv/all so:

<img width="1280" alt="Bildschirmfoto 2019-09-08 um 18 56 12" src="https://user-images.githubusercontent.com/42944977/64491581-70106600-d26a-11e9-94bd-76cc8e1cb17f.png">
